internal/abci/meta/meta.go

```diff
@@ -29,17 +35,30 @@ const (
 		`SET height = $1, app_hash = $2;`
 
 	getChainState = `SELECT height, app_hash FROM ` + chainSchemaName + `.chain;`
+
+	upsertParam = `INSERT INTO ` + chainSchemaName + `.consensus_params ` +
+		`VALUES ($1, $2) ` +
+		`ON CONFLICT (param_name) DO UPDATE SET param_value = $2;`
+
+	getParams = `SELECT param_name, param_value FROM ` + chainSchemaName + `.consensus_params;`
 )
 
 func initTables(ctx context.Context, tx sql.DB) error {
 	_, err := tx.Execute(ctx, initChainTable)
+	if err != nil {
+		return err
+	}
 	return err
```

Comment:
This function's error handing looks pretty odd now.  No conditional is required here, but if you want to keep an consistent approach, maybe change the second one to `return nil`, but personally it's too verbose.

Reply:
Oh whoops. I had made a change to this function and then undid it